### PR TITLE
Add tables: scheduled_payments, scheduled_payment_attempts

### PR DESCRIPTION
**Pull request by jurgenwerk**
Tickets: CS-4371, CS-4372

### CODE REVIEW

**Comment by lukemelia** [comment]
Let's add the new fee fields before me merge this.

File: packages/hub/db/migrations/20220810123029047_create-scheduled-payment-attempts.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { MigrationBuilder } from 'node-pg-migrate';
+
+const TABLE = 'scheduled_payment_attempts';
+const STATUS_ENUM = `${TABLE}_status_enum`;
+
+export async function up(pgm: MigrationBuilder): Promise<void> {
+  pgm.createType(STATUS_ENUM, ['in_progress', 'finished', 'failed']);
```
Comment (jurgenwerk) [inline packages/hub/db/migrations/20220810123029047_create-scheduled-payment-attempts.ts:7]: Note to self: actually use the enum 😅

File: packages/hub/db/migrations/20220810123016866_create-scheduled-payments.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    execution_gas_estimation: { type: 'bigint', notNull: true },
+    max_gas_price: { type: 'bigint', notNull: true },
+    fee_fixed_usd: { type: 'integer', notNull: true },
+    fee_percentage: { type: 'integer', notNull: true },
+    nonce: { type: 'bigint', notNull: true },
+    pay_at: { type: 'timestamp' },
+    recurring_day_of_month: { type: 'integer' },
+    recurring_until: { type: 'timestamp' },
+    valid_for_days: { type: 'integer', default: 3 },
+    sp_hash: { type: 'string', notNull: true },
```
Comment (jurgenwerk) [inline packages/hub/db/migrations/20220810123016866_create-scheduled-payments.ts:22]: Note to self: add uniqueness constraint

**Review by backspace — COMMENTED** [review]
The migrations look fine, is this meant to include the changes `structure.sql` file that come from actually running them? And it would make sense to then include the [Prisma changes](https://github.com/cardstack/cardstack/blob/main/packages/hub/README.md#database-migrations)

**Comment by jurgenwerk** [comment]
@backspace thanks - I added the new tables to the structure and the Prisma models as well. There are currently ongoing discussions whether we actually need the `nonce` (and if it would be good to constrain uniqueness by sender, recipient, amount, token, pay date (or recurring day in month)) so let's wait until we clarify that.

**Pull request merged**